---
layout: page
title: publications
description: Zhenxing Guo's publications
---

<div class="navbar">
    <div class="navbar-inner">
        <ul class="nav">
            <li><a href="#articles">articles</a></li>
                      <!-- <li><a href="#editorials">editorials</a></li> -->
                      <!-- <li><a href="#letters">letters</a></li> -->
            <li><a href="#chapters">chapters</a></li>
                      <!-- <li><a href="#techreports">tech reports</a></li> -->
                      <!-- <li><a href="#thesis">dissertation</a></li> -->
        </ul>
    </div>
</div>

### <a name="articles"></a>journal articles
**Methodology work**: 
#### MeRIP-seq
* D. Duan, W. Tang, R. Shu, **Z. Guo**, H. Feng* (2023+) Evaluation of Differential Epitranscriptome
Analysis Methods. _Briefings in Bioinformatics_, Under revision.
* **Z. Guo**, A. M. Shafik, P. Jin, H. Wu Differential RNA Methylation Analysis for MeRIP-seq Data under General Experimental Design. _Bioinformatics_, (2022). [[link]](https://doi.org/10.1093/bioinformatics/btac601)
* **Z. Guo**, Andrew M Shafik, Peng Jin, Zhijin Wu, Hao Wu, Detecting m6A methylation regions from Methylated RNA Immunoprecipitation Sequencing. _Bioinformatics_, (2021). [[link]](https://academic.oup.com/bioinformatics/advance-article-abstract/doi/10.1093/bioinformatics/btab181/6173980)

#### RNA-seq
* **Z Guo**, Y Cui, X Shi, JA Birchler, I Albizua, SL Sherman, ZS Qin, T Ji. An empirical bayesian approach for testing gene expression fold change and its application in detecting global dosage effects. _NAR Genomics and Bioinformatics_, (2020), 2(3):lqaa072. [[Link]]( https://academic.oup.com/nargab/article/2/3/lqaa072/5908375)
* Z Li, **Z Guo**, Y Cheng, P Jin, H Wu. Robust partial reference-free cell composition estimation from tissue expression. _Bioinformatics_, (2020), 36(11): 3431-3438. [[Link]]( https://academic.oup.com/bioinformatics/article/36/11/3431/5804977)

#### Non-negative Matrix Factorization
* **Z. Guo**, S Zhang. Sparse deep nonnegative matrix factorization. _Big Data Mining and Analytics_, (2019), 3(1):13-28.    
 [[Link]]( https://ieeexplore.ieee.org/document/8935092) 
 


**Collaborations**
* Y. Pan, K. Kathleen, **Z. Guo**, Y. Yang, G. Khalil, R. Kelly, B. Switzer (202x) Simulation Study for Improving Efficacy Estimates for New HIV Treatments. In preparation.
* M. C. Runnstrom1, A. Morrison-Porter, M. Ravindran, H. Quehl, R. Ramonell, M. Woodruff, M. Kuruvilla, F. Anam, R. Swerlick, C. Swenson, C. Polito, W. Neveu, R. Patel, C. Kim, l. Hentenaar, **Z. Guo**, H. Wu, N. S. Haddad, J. L. Daiss, I. Sanz, F. E. Lee*. Reduced SARS-CoV-2 Vaccine Response in Patients Treated with Biologic Therapies for Asthma. _Journal of Allergy and Clinical Immunology_, (2022), 149(2):AB319. [[link]](https://www.jacionline.org/article/S0091-6749(21)01844-3/fulltext)
* A. M. Shafik, F. Zhang, **Z. Guo**, Q. Dai, K. Pajdzik, Y. Li, Y. Kang, B. Yao, H. Wu, C. He, E. G. Allen, R. Duan, P. Jin. N6-methyladenosine dynamics in neurodevelopment and aging, and its potential role in Alzheimerâ€™s disease. _Genome Biology_, (2021), 22(1):1-19. [[link]](https://link.springer.com/article/10.1186/s13059-020-02249-z)

### <a name="chapters"></a>book chapters
* **Z. Guo**, A. Shafik, P. Jin, Z. Wu, H. Wu (2023) Analyzing mRNA Epigenetic Sequencing Data with TRESS. In: H. Oliveira, P. (eds) Computational Epigenomics and Epitranscriptomics. Methods in Molecular Biology, vol 2624. Humana, New York, NY. [[link]](https://doi.org/10.1007/978-1-0716-2962-8_12)
